src/kirby/components/slide-button/slide-button.component.scss

```diff
@@ -1,4 +1,6 @@
-@import "../../scss/web-imports";
+@import "web-imports";
+
+$slider-thumb-width: size('xxl');
```

Comment:
Use either `slide` or `slider` everywhere, dont mix

Reply:
Done

Reply:
This is coming from a native term "slider-thumb" that is not the same as the slide-button name we use here. So it would break styling if I renamed this slide-thumb everywhere. Otherwise, it should be slide-button everywhere.